### PR TITLE
Updated README with note to install requirements via make

### PR DESCRIPTION
**Pull request by clintonb**
- Updated mysql-connector-python to 1.2.3
- Updated README with flag for installing mysql-connector-python from an external source

@mulby @brianhw @rocha

### CODE REVIEW

File: requirements/default.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ filechunkio==1.5	# MIT
 html5lib==1.0b3 	# MIT
 httplib2==0.9 		# MIT
 mechanize==0.2.5	# BSD
-mysql-connector-python==1.2.2  	# GPL v2 with FOSS License Exception
+mysql-connector-python==1.2.3  	# GPL v2 with FOSS License Exception
```
Comment (rocha) [inline requirements/default.txt]: I think you can add the allow-external flag here:

```
--allow-external mysql-connector-python==1.2.3      # GPL v2 with FOSS License Exception
```

That means we can remove the comments above.

**Comment by rocha** [comment]
:rocket: check if you can use the external flag in the requirements file instead.

**Comment by clintonb** [comment]
@rocha `Makefile` actually includes the line. Should I update `README.md` reference `make requirements` instead of the pip command?

**Comment by rocha** [comment]
Well, if you add it the flag to the requirements file then you only have to keep it in one place instead of two.

**Comment by clintonb** [comment]
Updated.

**Comment by mulby** [comment]
Why do we need this change?

**Comment by clintonb** [comment]
Only the README change is needed/desired. I was initially under the impression that 1.2.2 is no longer available as I did not see it on pypi, however that doesn't seem to be the case.

**Comment by rocha** [comment]
Oh for some reason I thought it was related to the db update.

**Comment by clintonb** [comment]
I updated the commit to only modify the README.

**Comment by rocha** [comment]
why not the requirements file ? :disappointed:

**Pull request merged**